src/common/hotbar-store.injectable.ts

```diff
@@ -2,12 +2,377 @@
  * Copyright (c) OpenLens Authors. All rights reserved.
  * Licensed under MIT License. See LICENSE in root directory for more information.
  */
+
 import { getInjectable } from "@ogre-tools/injectable";
-import { HotbarStore } from "./hotbar-store";
+import catalogCatalogEntityInjectable from "./catalog-entities/general-catalog-entities/implementations/catalog-catalog-entity.injectable";
+import { action, comparer, observable, makeObservable, computed } from "mobx";
+import { BaseStore } from "./base-store";
+import migrations from "../migrations/hotbar-store";
+import { toJS } from "./utils";
+import { CatalogEntity } from "./catalog";
+import logger from "../main/logger";
+import { broadcastMessage } from "./ipc";
+import {
+  defaultHotbarCells,
+  getEmptyHotbar,
+  Hotbar,
+  CreateHotbarData,
+  CreateHotbarOptions,
+} from "./hotbar-types";
+import { hotbarTooManyItemsChannel } from "./ipc/hotbar";
+import type { GeneralEntity } from "./catalog-entities";
+
+export interface HotbarStoreModel {
+  hotbars: Hotbar[];
+  activeHotbarId: string;
+}
+
+interface Dependencies {
+  catalogCatalogEntity: GeneralEntity;
+}
+
+export class HotbarStore extends BaseStore<HotbarStoreModel> {
```

Comment:
I know that having the implementation beside the injectable is good, but for such a large implementation as this I don't think it is. I think that this should stay in `hotbar-store.ts`

Reply:
Removed all global shared state usages of HotbarStore since it wasn't that big thing. Implementation is in separate file again.